### PR TITLE
Issues/43

### LINKED ISSUE #43: Provide a way to train NLC for Object Storage search from Hubot
This involves integrating features from https://github.com/ibm-cloud-solutions/hubot-ibmcloud-objectstorage-crawler to scan and train for NLC.

### PR DESCRIPTION
**Pull request by kevan-ibm**
Fixes #43

### CODE REVIEW

File: src/scripts/object.index.js [hunk 1]
```diff
@@ -69,7 +69,20 @@ module.exports = (robot, res) => {
 			return;
 		}
 
-		env.searchEngine.index()
+		let commandContext = {
+			robot: robot,
+			res: res
+		};
+
+		let callback = function(callbackContext, error) {
```
Comment (kurtismgit) [inline src/scripts/object.index.js:77]: seems like you should check for if(error) and emit a message to user telling them there was a problem and they should consult logs.

File: src/scripts/object.scan.status.js [hunk 2]
```diff
@@ -70,11 +70,17 @@ module.exports = (robot, res) => {
 		}
 
 		env.searchEngine.getEngineStatus()
-			.then((indexResult) => {
-				console.log(indexResult);
+			.then((scanStatusResult) => {
+				console.log(scanStatusResult);
```
Comment (kurtismgit) [inline src/scripts/object.scan.status.js]: don't think you should have a console.log.  rather a robot.logger.debug

Comment (kevan-ibm) [reply src/scripts/object.scan.status.js]: Removed.

File: src/scripts/object.index.js (hunk 1 above)
Comment (kevan-ibm) [reply src/scripts/object.index.js:77]: Followed suggestion.

**Pull request merged**